src/agent/rustjail/src/container.rs

```diff
@@ -865,10 +870,8 @@ impl BaseContainer for LinuxContainer {
         let _ = fcntl::fcntl(pwfd, FcntlArg::F_SETFD(FdFlag::FD_CLOEXEC))
             .map_err(|e| warn!(logger, "fcntl pwfd FD_COLEXEC {:?}", e));
 
-        defer!({
```

Comment:
these `close()` calls can be deleted safely?

Reply:
The PipeStream will close the fd when dropped